src/main/java/tools/jackson/databind/introspect/JacksonAnnotationIntrospector.java

```diff
@@ -698,6 +711,15 @@ private List<NamedType> findSubtypesCheckRepeatedNames(String annotatedTypeName,
 
         return result;
     }
+    
+    private List<NamedType> findSubtypesByPermittedSubclasses(MapperConfig<?> config, Annotated a, Class<?> clazz)
```

Comment:
Add `// @since 2.19` for new/changed methods.

Reply:
This is targeted at master so it's since 3.0

Reply:
My bad, yes.